pad_crm/__manifest__.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2016 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+# Copyright 2019 Artem Rafailov <https://it-projects.info/team/Ommo73/>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+{
+    "name": """Pad on Leads(Opportunities)""",
+    "summary": """Collaborative editing Internal notes + history of changes""",
+    "category": "Customer Relationship Management",
+    # "live_test_url": "http://apps.it-projects.info/shop/product/DEMO-URL?version=12.0",
+    "images": [],
+    "version": "12.0.1.0.0",
+    "application": False,
+
+    "author": "IT-Projects LLC, Ivan Yelizariev",
+    "support": "apps@it-projects.info",
+    "website": "https://it-projects.info/",
+    "license": "LGPL-3",
+    "currency": "EUR",
```

Comment:
Where is the price attribute? If it's not set then add commented one like in the templates